### ISSUE TITLE
Webcam example don't work because of https

### ISSUE THREAD

**Issue by jcubic** [ejci/favico.js#141]
In Google Chrome to use a webcam you now need to serve the website over https.

**Comment by kylebakerio** [comment]
Came here to report this (that webcome isn't working, didn't look in to reason). Chrome 64 here.

**Comment by felix2056** [comment]
I really don't see how this is related to the plugin. Normally browsers like chrome and firefox will never allow non-https sites to run webrtc after Chrome 47+ version. This is their secure origin policy.
Although you can do some hack for this with some changes in ngnix.cong file but am not sure you'd want to go that far.

**Comment by kylebakerio** [comment]
the point is the demo/example is broken, and needs to be updated. 

Obviously this is how Chrome works. The example just hasn't been updated since before chrome had that requirement.

**Comment by felix2056** [comment]
@kylebakerio if that's the case then instead of waiting for the fix, you can install the free let's encrypt certificate on your server, it'll only take 5 minutes and lasts for 90 days.
Renewal is also free

**Comment by jcubic** [comment]
Yes the issue is about demo that runs on http (see the title of the issue), it's not the code itself or your website that use the library.

**Comment by kylebakerio** [comment]
You're not hearing us, felix. Stop responding like we're idiots, take a second, and read what is being written here. We're not talking about a problem the repo code has. We're talking about the demo the repo is using to advertise itself.

**Comment by kylebakerio** [comment]
http://lab.ejci.net/favico.js/

"start webcam"

in console:

```js
favico.min.js:7 Uncaught Error: Error setting webcam. Message: navigator.getUserMedia is not a function
    at w (favico.min.js:7)
    at Object.L [as webcam] (favico.min.js:7)
    at HTMLButtonElement.<anonymous> (example.js:127)
    at HTMLButtonElement.dispatch (jquery-1.10.1.min.js:5)
    at HTMLButtonElement.v.handle (jquery-1.10.1.min.js:5)
```

**Comment by kylebakerio** [comment]
Also, most of the demos are broken in general. Several work, but many don't.
